src/test/resources/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs/AS_Annotations.gatk3.7_30_ga4f720357.expected.vcf

```diff
@@ -133,29 +133,28 @@
 ##contig=<ID=GL000192.1,length=547496>
 ##contig=<ID=NC_007605,length=171823>
 ##reference=file:///seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta
-##source=GenotypeGVCFs
 #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA12878
-20	10087820	.	C	CAG	294.64	.	AC=1;AF=0.500;AN=2;AS_BaseQRankSum=-2.100;AS_FS=2.811;AS_MQ=60.00;AS_MQRankSum=0.600;AS_QD=6.15;AS_ReadPosRankSum=-0.400;AS_SOR=1.147;BaseQRankSum=-2.273e+00;DP=93;ExcessHet=3.0103;FS=4.432;MLEAC=1;MLEAF=0.500;MQ=59.55;MQRankSum=0.678;QD=6.14;RAW_MQ=329810.00;ReadPosRankSum=-4.920e-01;SOR=1.302	GT:AD:GQ:PL	0/1:35,13:99:302,0,1079
```

Comment:
I understand why the HaplotypeCaller outputs changed at SpanningDeletion sites. What about these files? This one looks like it has neither spanning deletions to genotype nor has its input file changed.  Can you explain why there was so much jitter with the old outputs?

Reply:
No, I have no idea. The behavior of HaplotypeCaller must have changed slightly over time to create this jitter without a corresponding update to the integration test files. These changes of e.g. 0.03 or 0.04 to qual score didn't cause the test to fail, I think given the `DEFAULT_FLOAT_TOLERANCE` in `BaseTest`. Since I added automatic update of expected test results to the `GenotypeGVCFsIntegrationTest`, the jitter was passed along to the expected files in this PR. If there's a historical reason that we need to preserve these test files exactly as is, I can revert the automatic-update feature in the test and manually update only the files that have spanning deletion in them -- otherwise I think updating them to account for current jitter is the price of implementing an auto-update.

Reply:
Ahh that makes a lot of sense. In that case 👍